Adds abandoned lab as an exoplanet spawn

DeadmonWonderland opened the pull request:
:cl: Deadmon
rscadd: Adds an abandoned lab as an exoplanet spawn.
:cl:

Spookerton commented on maps/random_ruins/exoplanet_ruins/oldlab/oldlab.dm:
```diff
@@ -0,0 +1,13 @@
+	cost = 1
+	template_flags = TEMPLATE_FLAG_CLEAR_CONTENTS | TEMPLATE_FLAG_NO_RUINS
+	ruin_tags = RUIN_HUMAN
+	apc_test_exempt_areas = list(
```
already an empty list, no need to redefine it

DeadmonWonderland replied:
done

Spookerton commented on maps/random_ruins/exoplanet_ruins/oldlab/oldlab.dm:
```diff
@@ -0,0 +1,13 @@
+#include "oldlabareas.dm"
```
double include; also probably preferable to put the areas in oldlab.dm instead of a separate file.

Spookerton commented on baystation12.dme:
```diff
@@ -3186,6 +3186,9 @@
 #include "maps\random_ruins\exoplanet_ruins\monoliths\monoliths.dm"
 #include "maps\random_ruins\exoplanet_ruins\oasis\oasis.dm"
+#include "maps\random_ruins\exoplanet_ruins\oldlab\oldlab.dm"
+#include "maps\random_ruins\exoplanet_ruins\oldlab\oldlabareas.dm"
```
double include (see review above)

DeadmonWonderland replied:
fixed

Spookerton commented on baystation12.dme:
```diff
@@ -3186,6 +3186,9 @@
 #include "maps\random_ruins\exoplanet_ruins\oasis\oasis.dm"
+#include "maps\random_ruins\exoplanet_ruins\oldlab\oldlab.dm"
+#include "maps\random_ruins\exoplanet_ruins\oldlab\oldlabareas.dm"
+#include "maps\random_ruins\exoplanet_ruins\oldlab\oldlab.dmm"
```
don't include submap maps directly

DeadmonWonderland replied:
done

DeadmonWonderland committed:
Adds old lab

Adds an old lab that spawns on exoplanets, includes it's own atmos system and working airlock.

Final Final
super final
final
affa
final
